mandatory/9-group-people.js

```diff
@@ -0,0 +1,33 @@
+/*
+  Write a function that accepts two arrays as parameters:
+  1. An array of volunteer names
+  2. An array of trainee names
+
+  The function should return a single array containing *both* the volunteers and trainee names.
+*/
+
+function groupPeople() {}
+
+/* 
+===================================================
+======= TESTS - DO NOT MODIFY BELOW THIS LINE =====
+There are some Tests in this file that will help you work out if your code is working.
+To run the tests for just this one file, type `npm test -- --testPathPattern 2-oxygen-levels` into your terminal
+(Reminder: You must have run `npm install` one time before this will work!)
+===================================================
+*/
+
+TestScheduler("volunteersAndTrainees works", () => {
```

Comment:
I'm assuming this is an auto-complete error?

Reply:
Yep, my vscode has this annoying habit of auto-selecting the first suggestion and I'm not sure why. Will fix.